tests/functional/spec/features/supplementary_data/supplementary_data.spec.js

```diff
@@ -370,11 +370,11 @@ describe("Using supplementary data", () => {
   it("Given there is now an additional product, When I resume the Product Details Section, Then I start from the list collector content block and see the new product is incomplete", async () => {
     await $(HubPage.summaryRowLink("section-6")).click();
     await expect(browser).toHaveUrlContaining(ListCollectorProductsPage.pageName);
-    await checkListItemComplete(`dt[data-qa="list-item-1-label"]`);
-    await checkListItemComplete(`dt[data-qa="list-item-2-label"]`);
-    await checkListItemIncomplete(`dt[data-qa="list-item-3-label"]`);
+    await summaryItemComplete(`li[data-qa="list-item-1-label"]`);
+    await summaryItemComplete(`li[data-qa="list-item-2-label"]`);
+    await summaryItemIncomplete(`li[data-qa="list-item-3-label"]`);
```

Comment:
This needs to call the list item method? Hence the failures?

Reply:
Good spot, missed that one